spec/index.html

```diff
@@ -239,9 +249,15 @@
 
       <section id="abstract">
         <p>
-          This specification describes Map Markup Language (MapML), which is a hypertext document format for maps.
-          This specification also extends the semantics of the standard [[HTML]] 
-          <a href="https://html.spec.whatwg.org/multipage/image-maps.html#the-map-element"><code>map</code></a> element.
+          This specification describes Map Markup Language (MapML), which is an extended subset of HTML, for maps.
+          MapML extends the semantics of several  [[[HTML]]] elements, and specifies a small set of new, mapping-specific 
+          elements, in the HTML namespace ("http://www.w3.org/1999/xhtml/"). 
+        </p>
+
+        <p>MapML is designed to specify a declarative markup vocabulary that "implements" some of the capabilities 
+          laid out by the Use Cases and Requirements for Web Maps specification.  As capabilities are identified, 
```

Comment:
Maybe make "Use Cases and Requirements for Web Maps" into a link instead? (`[[[UCR-Web-Maps]]]`)

Reply:
okey doke